Fineo/Tests/Test.FileStorage.AzreBlob/TestFileStorage.cs

```diff
@@ -0,0 +1,305 @@
+﻿using Fineo.AzureEmulatorHelper;
+using Fineo.Interfaces;
+using Microsoft.Extensions.Configuration;
+using Microsoft.WindowsAzure.Storage;
+using Newtonsoft.Json;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Test.FileStorage.AzureBlob
+{
+    public class TestFileStorage
+    {
+        class FileStorageConfig
+        {
+            [JsonProperty("StorageAccountName")]
+            public string StorageAccountName { get; set; }
+
+            [JsonProperty("StorageAccountKey")]
+            public string StorageAccountKey { get; set; }
+
+            [JsonProperty("ContainerName")]
+            public string ContainerName { get; set; }
+
+            [JsonProperty("BlobEndpoint")]
+            public string BlobEndpoint { get; set; }
+
+        }
+
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            AzureStorageEmulator ase = new AzureStorageEmulator();
+            ase.Start();
+
+            // letting emulator start
+            Task.Delay(5000).Wait();
+        }
+
+        [Test]
+        public void InitFileStorage_Success()
```

Comment:
Add scenarios for:
- getting file info
- getting list of items - files & subdirs - in folder